app/views/directives/edit-environment-from.html

```diff
@@ -0,0 +1,94 @@
+<ng-form name="forms.editEnvironmentFrom" novalidate>
```

Comment:
Should this be `$ctrl.editEnvironmentFrom` here and below?

Reply:
Is that a vote to remove the ```ng-form```? 

I may be able to, but was under the impression we were trying to maintain the parent-child relationship with the parent form?

Reply:
You should be able to keep `ng-form` and use `$ctrl.editEnvironmentForm` I think. Is this sharing the same form object as the parent? I don't think that should be necessary.